app/views/whatDoYouWantToDo.scala.html

```diff
@@ -19,14 +19,17 @@
 @import uk.gov.hmrc.tai.config.ApplicationConfig
 @import uk.gov.hmrc.tai.viewModels.WhatDoYouWantToDoViewModel
 @import includes.forms.inputRadioGroupNewStyling
+@import uk.gov.hmrc.play.language.LanguageUtils.Dates
+@import uk.gov.hmrc.time.TaxYearResolver
 
 @(wdywtdForm: Form[uk.gov.hmrc.tai.forms.WhatDoYouWantToDoFormData], model: WhatDoYouWantToDoViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.TaiUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
 
 @radioSequence = @{
-    val mandatory = Seq(("currentTaxYear" -> Messages("tai.WhatDoYouWantToDo.radio1")), ("lastTaxYear" -> Messages("tai.WhatDoYouWantToDo.radio2")) )
-    if(model.isCyPlusOneEnabled) { mandatory :+ ("nextTaxYear" -> Messages("tai.WhatDoYouWantToDo.radio3")) } else { mandatory }
+    val mandatory = Seq(("currentTaxYear" -> Messages("tai.WhatDoYouWantToDo.radio2",s"${Dates.formatDate(TaxYearResolver.startOfCurrentTaxYear)}",s"${Dates.formatDate(TaxYearResolver.endOfCurrentTaxYear)}")), ("lastTaxYear" -> Messages("tai.WhatDoYouWantToDo.radio3")) )
+    if(model.isCyPlusOneEnabled) { ("nextTaxYear" -> Messages("tai.WhatDoYouWantToDo.radio1")) +: mandatory  } else { mandatory }
```

Comment:
Think we should keep the radio numbers sequential in terms message keys - 1, 2 mandatory, & 3 optional